### PR TITLE
lesson-4-1

### PR DESCRIPTION
**Pull request by Manizha-khorram**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,18 +1,49 @@
...
-        <a  href="https://github.com/Manizha-khorram">GitHub account</a> <br>
-        <a  href="https://www.linkedin.com/in/manizha-khorram/">LinkedIn account</a> 
+        <img src="Developer Girl.jpeg" alt="Picture of a girl which is coding" id="position">
+        <section>
+            <a href="#About"><h1>About</h1> </a>
+            <p>I am from Afghanistan and I love coding. I would like to work as professional sofware developer. I hold bachelor of computer science degree, I love problem solving and creating new things.I keen to learn challenging roles in javaScript, software development and related technologies.</p>
+        </section>
+
+        <section>
+            <a href="#Experience"><h1>Experience</h1></a>
```
Comment (p34c310v3) [inline index.html:30]: Just a side note on using more than one `<h1>` on a page:

> Using more than one `<h1>` is allowed by the HTML specification, but is not considered a best practice. Using only one `<h1>` is beneficial for screenreader users.

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/Heading_Elements

Comment (p34c310v3) [reply index.html:30]: I also found this article on "Link in Header? Or Header in Link?":
https://css-tricks.com/link-header-header-link/

:smile:

File: index.html [hunk 2]
```diff
@@ -1,18 +1,49 @@
...
+            <p>I have worked as a system development analyst at NPA (National Procurement Authority) and USWDP (University Work Force Development Program) in Afghanistan.</p>
+        </section>
+
+        <section>
+            <a href="#Connect"> <h2>Connect</h2> </a>
+            <ul>
+              <li>  <a  href="mailto:manizh.khorram@gmail.com">Gmail</a>  </li><br>
+              <li> <a  href="https://github.com/Manizha-khorram">GitHub account</a>  </li> <br>
```
Comment (p34c310v3) [inline index.html:38]: I don't think the `<br>` tag is necessary at the end of this list item. There also seems to be some extra white space and some formatting inconsistencies here and there. :smile: :sweat_smile:

File: index.html [hunk 3]
```diff
@@ -1,18 +1,49 @@
...
     <body>
-        <h2>About</h2>
-        <p>I am from Afghanistan and I love coding. I would like to work as professional sofware developer. I hold bachelor of computer science degree, I love problem solving and creating new things.I keen to learn challenging roles in javaScript, software development and related technologies.</p>
-        <h2>Experience</h2>
-        <p>I have worked as a system development analyst at NPA (National Procurement Authority) and USWDP (University Work Force Development Program) in Afghanistan.</p>
-        <h2>Connect</h2>
-        <a  href="mailto:manizh.khorram@gmail.com">Gmail</a> <br>
-        <a  href="https://github.com/Manizha-khorram">GitHub account</a> <br>
-        <a  href="https://www.linkedin.com/in/manizha-khorram/">LinkedIn account</a> 
+        <img src="Developer Girl.jpeg" alt="Picture of a girl which is coding" id="position">
```
Comment (p34c310v3) [inline index.html:23]: I like that you provided an `alt` attribute! :100: :raised_hands: Also, great image! :+1: :smile:

File: index.html [hunk 4]
```diff
@@ -1,18 +1,49 @@
...
+        <section>
+            <a href="#About"><h1>About</h1> </a>
+            <p>I am from Afghanistan and I love coding. I would like to work as professional sofware developer. I hold bachelor of computer science degree, I love problem solving and creating new things.I keen to learn challenging roles in javaScript, software development and related technologies.</p>
+        </section>
+
+        <section>
+            <a href="#Experience"><h1>Experience</h1></a>
+            <p>I have worked as a system development analyst at NPA (National Procurement Authority) and USWDP (University Work Force Development Program) in Afghanistan.</p>
```
Comment (p34c310v3) [inline index.html:31]: :+1: :sunglasses:

File: index.html [hunk 5]
```diff
@@ -1,18 +1,49 @@
...
-        <h2>Experience</h2>
-        <p>I have worked as a system development analyst at NPA (National Procurement Authority) and USWDP (University Work Force Development Program) in Afghanistan.</p>
-        <h2>Connect</h2>
-        <a  href="mailto:manizh.khorram@gmail.com">Gmail</a> <br>
-        <a  href="https://github.com/Manizha-khorram">GitHub account</a> <br>
-        <a  href="https://www.linkedin.com/in/manizha-khorram/">LinkedIn account</a> 
+        <img src="Developer Girl.jpeg" alt="Picture of a girl which is coding" id="position">
+        <section>
+            <a href="#About"><h1>About</h1> </a>
+            <p>I am from Afghanistan and I love coding. I would like to work as professional sofware developer. I hold bachelor of computer science degree, I love problem solving and creating new things.I keen to learn challenging roles in javaScript, software development and related technologies.</p>
```
Comment (p34c310v3) [inline index.html:26]: :heart: :raised_hands:

**Review by p34c310v3 — APPROVED** [review]
I left a few comments, but overall I think it's pretty good! :+1: :clap: :raised_hands:

**Comment by Manizha-khorram** [comment]
It is really nice of you that reviewed and commented on my assignment, you mentioned great points and I would try to consider the points for my next assignments, and the link for "Link in Header? Or Header in Link?" is thinkable point which was useful, thanks😄.

**Pull request merged**